src/main/java/com/jnape/palatable/lambda/adt/hlist/Tuple8.java

```diff
@@ -74,6 +74,16 @@
         return new HCons<>(_0, this);
     }
 
+    /**
+     * Snoc an element onto the back of this HList.
+     *
+     * @param _9   the new last element
+     * @return the updated HList
+     */
+    public <_9> HCons<_1, Tuple8<_2, _3, _4, _5, _6, _7, _8, _9>> snoc(_9 _9) {
```

Comment:
Let's kill this particular one off for now so we're still only offering constant time operations on tuples. Calling `snoc` on a `Tuple8` seems like a rare enough use case that I'm alright asking people to do it themselves, and `Tuple8#into((_1, _2, _3, _4, _5, _6, _7, _8) -> tuple(_2, _3, _4, _5, _6, _7, _8, _9).cons(_1))` doesn't seem unreasonable to me.